doc/help-search.md

```diff
@@ -20,6 +20,12 @@
   - `topic:topic-name`: Searches for packages that have specified the
     `topic-name` [topic](/topics).
 
+  - `publisher:publisher-name.com`: Searches for packages published by  `publisher-name.com`
```

Comment:
Could we do:

```suggestion
  - `publisher:<publisher-name.com>`: Searches for packages published by  `publisher-name.com`
```

(And similar in the other examples (also above). That would be easier to read IMO.

Reply:
Good idea! Otherwise it's not completely clear what part is the changeable argument. Plus that would follow how we document pub command arguments.